content/beginner/110_irsa/preparation.md

```diff
@@ -1,44 +1,50 @@
 ---
 title: "Preparation"
-date: 2018-11-13T16:36:24+09:00
+date: 2021-07-20T00:00:00-03:00
 weight: 10
 draft: false
 ---
 
 ### Enabling IAM Roles for Service Accounts on your Cluster
 
-* The IAM roles for service accounts feature is available on new Amazon EKS Kubernetes version 1.14 clusters, and clusters that were updated to versions 1.14 or 1.13 on or after September 3rd, 2019.
+* The IAM roles for service accounts feature is available on new Amazon EKS Kubernetes version 1.16 or higher, and clusters that were updated to versions 1.14 or 1.13 on or after September 3rd, 2019.
 
 {{% notice info %}}
-If your EKS cluster version is lower or does not match with above, please read the [updating an Amazon EKS Cluster](https://docs.aws.amazon.com/eks/latest/userguide/update-cluster.html) section in the User Guide.
+If your EKS cluster version is older than 1.16 your outputs may very. Please consider reading the [updating an Amazon EKS Cluster](https://docs.aws.amazon.com/eks/latest/userguide/update-cluster.html) section in the User Guide.
 {{% /notice %}}
 
 ```bash
 kubectl version --short
 ```
 
+Output:
+{{< output >}}
+Client Version: v1.20.4-eks-6b7464
+Server Version: v1.20.4-eks-6b7464
+{{< /output >}}
+
 {{% notice info %}}
-If your aws cli version is lower than 1.18.15, use [Installing the AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html) in the User Guide
+If your aws cli version is lower than 1.20.3, use [Installing the AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html) in the User Guide
```

Comment:
@davivcgarcia this item needs to be updated to 1.19.122 as well.

Reply:
Done!